HomeWork06/task3.py

```diff
@@ -0,0 +1,34 @@
+# використати built-in функцію
+
+collection = [i for i in range(1, 999999)]
+the_greatest_number = max(collection)
+
+
+# створити свою функцію
+def maximum(numbers):
+    biggest_number = 0
+    for number in numbers:
+        if number > biggest_number:
+            biggest_number = number
+    return biggest_number
+
+
+# використати лямбда функцію
+
+# Need your comments on the details of this task as far as I'm not sure whether my solution is correct, and I've got the
+#  general point of this task. First thought about using a for loop in a lambda function to iterate over each element in
+#  collection and further to compare it with some variable (same as in the above-shown "maximum" function), but in one
+# line lambda function, but discovered that we can't use "for" statement in lambda, so the only way I see here to solve
+# this task is to use my "maximum" function or built-in "max" function. Please let me know if there is more true python
+# solution to this exercise. The point is that we are trying to figure out the greater value inside a collection, so we
+# have to iterate. I can't see the possibility to reproduce lambda a, b: a if a > b else b over the iterable.
+
+maximum_lambda = lambda some_collection, some_function: some_function(some_collection)
```

Comment:
actually, the task was about this solution, to understand how to work with lambdas :)